front/app/src/js/components/game/Engine/_GameSocketIO.js

```diff
@@ -86,167 +66,226 @@ export class _GameSocketIO {
       console.log('Server debug message: ', message);
     });
 
-    this.#socketIO.on('sync_time', (serverTime) => {
-      ServerTimeFixer.updateServerLatency(serverTime);
+    this.#socketIO.on('time_sync', (timeData) => {
+      ServerTime.syncServerTime(timeData);
     });
 
```

Comment:
New event to sync the time between server and client. Server sends it after the client emits `request_time_sync`